src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -830,14 +855,11 @@ def main(argv):
         '%s_%d' % (strategy.FORK_STRATEGY, num_fuzz_processes))
 
   extra_env = {}
-  if do_mutator_plugin():
-    mutator_plugin_path = mutator_plugin.get_mutator_plugin(target_name)
-    if mutator_plugin_path:
-      logs.log('Using mutator plugin: %s' % mutator_plugin_path)
-      # TODO(metzman): Change the strategy to record which plugin was used, and
-      # not simply that a plugin was used.
-      fuzzing_strategies.append(strategy.MUTATOR_PLUGIN_STRATEGY)
-      extra_env['LD_PRELOAD'] = mutator_plugin_path
+  if use_minijail:
```

Comment:
Why need this when minijail_chroot already has the right value:

  if use_minijail:
    minijail_chroot = runner.chroot
  else:
    minijail_chroot = None

Reply:
Done.